src/components/python-pages/Lesson.js

```diff
@@ -63,6 +63,13 @@ class Lesson extends React.Component {
         const paragraphs = container.querySelectorAll("p");
         paragraphs.forEach((paragraph) => { paragraph.classList.add("p-body-medium"); });
 
+        const images = container.querySelectorAll("img");
+        images.forEach((image) => { 
+          image.style.minWidth = "200px"; 
+          image.style.height = "auto"; 
+          image.style.width = "40%";
```

Comment:
Should we have a set width?  I don't mind setting the width when making the lesson in the admin page because I might want some images to be 30%, some others to be 40%, etc.  I think just remove the set width style?